llvm/test/CodeGen/RISCV/pr114316.ll

```diff
@@ -0,0 +1,53 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -riscv-v-vector-bits-max=512 -mtriple=riscv64 -mattr=+m,+c,+v -O0 < %s | FileCheck --check-prefix=RV64V %s
+
+; Function Attrs: nocallback nofree nosync nounwind willreturn
```

Comment:
Can you reduce the attributes (and remove the pointless Function Attrs comments)? I imagine you don't need most of them.

Reply:
addressed

Reply:
Thanks, looks much better now (without actually digging into the details of the change)